src/extension.ts

```diff
@@ -148,6 +148,9 @@ export class Extension {
         }
         await this._reusedBrowser.inspect(this._models);
       }),
+      vscode.commands.registerCommand('pw.extension.command.closeBrowsers', () => {
+        this._reusedBrowser.stop();
```

Comment:
Does VSCode allow you to hook into close lifecycle of the VSCode itself and trigger this there too? I got in state where I had closed VSCode and then I had to close browser after manually.

Reply:
That would be very strange - I thought this part of the pipeline is supersolid. When VSCode goes, driver self-desctructs, taking all the browsers with itself.